dss/stepfunctions/checkout/checkout_states.py

```diff
@@ -1,13 +1,48 @@
+retry_config = [
+    {
+        "ErrorEquals": ["States.TaskFailed"],
+        "IntervalSeconds": 5,
+        "MaxAttempts": 5,
+        "BackoffRate": 1.5
+    },
+    {
+        "ErrorEquals": ["States.Timeout"],
+        "IntervalSeconds": 30,
+        "MaxAttempts": 3,
+        "BackoffRate": 1.5
+    },
+    {
+        "ErrorEquals": ["States.Permissions"],
+        "MaxAttempts": 0
+    },
+    {
+        "ErrorEquals": ["States.ALL"],
```

Comment:
Just a note: This will retry upon all exceptions. It's possible to retry on specific exceptions by name, e.g `"ErrorEquals": ["MyCustomException", "TypeError", "States.Timeout"],`

Reply:
this is a catch all. it would first match on the specific Errors, please see above